editor/js/Sidebar.Geometry.js

```diff
@@ -192,11 +192,7 @@ function SidebarGeometry( editor ) {
 
 		}
 
-		const left = ( screen.width - 500 ) / 2;
-		const top = ( screen.height - 500 ) / 2;
-
-		const url = URL.createObjectURL( new Blob( [ output ], { type: 'text/plain;charset=utf-8' } ) );
-		window.open( url, '_blank', `location=no,left=${left},top=${top},width=500,height=500` );
```

Comment:
@mrdoob Is there a special reason for showing the JSON in a new window? Maybe we overlook a use case here...